### PR TITLE
ILS Fixes

### LINKED ISSUE #806: LFLB
# Summary of Navigation issue. (ILS, Waypoints, etc.)

Movement of main runway. 

# Runway(s) with ILS (If any)

36-18

# References (Official sources/charts)

http://vau.aero/navdb/chart/LFLB.pdf

### LINKED ISSUE #905: OPLA
# Summary of Navigation issue: (ILS, Waypoints, etc.)

Runway moved to match imagery. ILS + Waypoints/Fixes

# Runway(s) with ILS: (If any)

36R 

# References: (Official sources/charts)

http://vau.aero/navdb/chart/OPLA.pdf

### LINKED ISSUE #1637: ZPLC
# Summary of Navigation issue: (ILS, Waypoints, etc.)

< ILS >

# Runway(s) with ILS: (If any)

< 16 >

# References: (Official sources/charts)

< 
ZPLCILS16
>

### LINKED ISSUE #1638: ZSLY
# Summary of Navigation issue: (ILS, Waypoints, etc.)

< ILS >

# Runway(s) with ILS: (If any)

< 19/01 >

# References: (Official sources/charts)

< 
ZSLYILS
 >

### LINKED ISSUE #1648: KMEM
# Summary of Navigation issue: (ILS, Waypoints, etc.)

ILS

# Runway(s) with ILS: (If any)

All Runways

# References: (Official sources/charts)

https://www.iflightplanner.com/Airports/KMEM

### PR DESCRIPTION
**Pull request by Czechoslovaki**
LFLB (Fixed #806)
OPLA (Fixed #905)
ZPLC (Fixed #1637)
ZSLY (Fixed #1638)
KMEM (Fixed #1648)

### CODE REVIEW

File: Glideslope.json [hunk 1]
```diff
@@ -43578,6 +43578,19 @@
...
+   "name": "GS",
+   "airportICAO": "ZPLC",
+   "identifier": "ILG",
+   "associatedRunwayNumber": "16",
+   "frequency": 108500,
+   "elevation": 6204,
+   "latitude": 23.74877778,   
+   "longitude": 100.01944371,
+   "bearing": 154.57,
+   "glideslope": 3.0001314,
```
Comment (AnthonyCK) [inline Glideslope.json:43591]: Hi! The glideslope should be 3.2° here, so need to change to 3.2015457
image

Comment (Czechoslovaki) [reply Glideslope.json:43591]: @AnthonyCK that looks to me a 3.2 degree runway downgrade 🤷

Comment (MajorTomMueller) [reply Glideslope.json]: Yes, this is a 3.2deg glideslope. But the separator is not taken into the GS.json the format for glideslope is three didgits GS and 5 digits bearing without any separation.
So 3.21 GS and 123.45 bearing becomes 32112345 in Gs.jason

Comment (Czechoslovaki) [reply Glideslope.json]: ok, then it seems the editor has made an error with the PAPI angle of the airport.

Comment (MajorTomMueller) [reply Glideslope.json]: Fixed in airports

**Pull request merged**